### ISSUE TITLE
Feature Request: Copy file to and from app

### ISSUE THREAD

**Issue by siuying** [appcelerator/node-ios-device#13]
It'd be helpful to add function to copy files to and from app Documents/Cache folder. 

This would help to automate process of copy logs/databases to/from devices.

**Comment by cb1kenobi** [comment]
Agreed! I could really use this functionality too. I'll add it to my radar and see about putting some time into it, but no promises. :)

**Comment by hansemannn** [comment]
This can be done as part of the new Vite-based build pipeline that @janvennemann has been working on. It will rather be part of https://github.com/tidev/titanium-sdk though. Thank you!
